plugins/discourse-local-dates/spec/system/local_dates_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+describe "Local dates", type: :system, js: true do
+  fab!(:topic) { Fabricate(:topic) }
+    fab!(:user) { Fabricate(:user) }
+    fab!(:post) { Fabricate(:post_with_local_dates, topic: topic) }
+
+    it "renders local dates and date ranges correctly" do
+      sign_in user
+
+      visit "/t/#{topic.id}"
+      topic_page = PageObjects::Pages::Topic.new
```

Comment:
This has a `visit_topic` helper already

Reply:
Also generally we have been storing these at the top of the spec in `let` blocks